src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,36 @@
 package core.basesyntax;
 
-public class Car {
+/**
+ * Don’t use super.clone().
+ * Use constructor to create new objects.
+ * Use fields itself instead of getters.
+ * Don’t use this. keyword. This is redundant.
+ * Don’t create redundant variables.
+ */
+
+public class Car implements Cloneable {
+    private Engine engine;
+    private Color color;
+    private double weight;
+    private String model;
+    private boolean isnew;
+
+    public Car() {
+        engine = new Engine();
+        color = Color.BLUE;
+        weight = 2500;
+        model = "BMV";
+        isnew = true;
```

Comment:
same as above

Reply:
Can randomize as above, but is any better way? Or i can just left it as is?